include/deal.II/base/tensor.h

```diff
@@ -166,6 +167,31 @@ class Tensor<0, dim, Number>
   constexpr DEAL_II_CUDA_HOST_DEV
   Tensor(const OtherNumber &initializer);
```

Comment:
Shouldn't this one be active unconditionally?

Reply:
Of course, it should. Hopefully fixed.